aptos-move/framework/aptos-framework/sources/vesting.move

```diff
@@ -662,7 +662,7 @@ module aptos_framework::vesting {
         contract_creation_seed: vector<u8>,
     ): (signer, SignerCapability) acquires AdminStore {
         let admin_store = borrow_global_mut<AdminStore>(signer::address_of(admin));
-        let seed = bcs::to_bytes(admin);
+        let seed = bcs::to_bytes(&signer::address_of(admin));
```

Comment:
Is this a bug? staking_contract also does something similar

Reply:
bug, not necessarily, just isn't great readability.